Applications/AlgorithmPlaza.qmd

```diff
@@ -50,46 +54,48 @@ The table below provides an overview of the different maturity levels that are a
+The **Overview** section presents profile details such as name, email, and affiliated organization. These details can be updated by clicking the `MANAGE` button. Meanwhile, in the **Team** sub-menu, a list of all organization members is displayed. This list can also be accessed by clicking the `VIEW TEAM` button at the bottom left of the overview sub-menu. Further information on team management can be found [here](PlazaDetails/ManageOrg.qmd). Lastly, within **Organisation** sub-menu, organization information such as name, email address, website, VAT number, and more can be viewed and modified.
+
 
 ###### Step 2: Manage your Organisation
 
-Each registered user acts as a personal organisation; you can modify its details as discussed in Step 1. However, this step is helpful when working with multiple individuals or being part of multiple organisations. Organisations allow collaboration with other users, facilitating the sharing of billing and service accounts. For more information on managing your organisation, please refer to the [organisation's documentation](PlazaDetails/ManageOrg.qmd) of the openEO Algorithm Plaza.
+Each registered user acts as a personal organization; details can be modified as discussed in Step 1. However, this step is helpful when working with multiple individuals or being part of multiple organizations. Organizations allow collaboration with other users, facilitating the sharing of billing and service accounts. For more information on managing the organization, please refer to the [organisation's documentation](PlazaDetails/ManageOrg.qmd) of the openEO Algorithm Plaza.
 
 ###### Step 3: Check your Credits
 
 Using any openEO processes, including those offered as services in this marketplace, consumes a certain amount of credits. Credits are crucial in executing openEO processes, serving as the main currency for accessing services and processing resources. Notably, these credits are shared among organisations. Whenever a service or a supported processing platform is executed, credits from the shared pool cover the resource consumption.
 
-This marketplace simplifies credit management, allowing users to monitor their account's credits easily. You can check your openEO credit under the [`Billing`](https://marketplace-portal.dataspace.copernicus.eu/billing){target="_blank"} section. Moreover, every user is provided with 4000 credits each month, with which they can execute multiple services.
+This marketplace simplifies credit management, allowing users to monitor their account's credits easily. The credit can be monitored credit under the [`Billing`](https://marketplace-portal.dataspace.copernicus.eu/billing){target="_blank"} section. Moreover, every user is provided with 4000 credits each month, with which they can execute multiple services.
```

Comment:
the credit can be monitored credit; please revise this sentence

Reply:
maybe write 4000 free credits? And are these openEO credits or cdse credits? 

How does it work with sentinel hub?

Reply:
These are openEO credits. For sentinel hub and CF there is a seperate set of free packages as shown here https://documentation.dataspace.copernicus.eu/Quotas.html